### ISSUE TITLE
Error when combining `bind` and external command flag `<` in `TIG_SCRIPT`

### ISSUE THREAD

**Issue by guludo** [jonas/tig#1357]
I'm unable to use the `bind` command combined with `<` to exit tig after external command when using `TIG_SCRIPT`. Here is an example:

```console
$ TIG_SCRIPT=<(echo ':bind generic x <echo hello') tig
tig: Error reading script: Missing '>' from key mapping: <echo hello
```

The same `:bind generic x <echo hello` command works fine if executed from the tig ui.

**Comment by koutcher** [comment]
The way TIG_SCRIPT is implemented, you have to use `TIG_SCRIPT=<(echo ':bind generic x <lt>echo hello') tig`.

**Comment by koutcher** [comment]
No, I deducted it from the implementation. It's not straightforward, we'll add a note to the documentation. The content of the script file is actually interpreted as keytrokes (TIG_SCRIPT was introduced for the test suite), so to get a `<` you need to press the corresponding key, and as `<` is reserved to specify key values, you have no other choice than specifying its full key value, so <LT>.
